src/Testcontainers.EventStore/EventStoreBuilder.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Testcontainers.EventStore
+{
+    public class EventStoreBuilder : ContainerBuilder<EventStoreBuilder, EventStoreContainer, EventStoreConfiguration>
+    {
+        private const string EventStoreImage = "eventstore/eventstore:21.2.0-buster-slim";
+
+        public const ushort EventStorePort = 2113;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+        /// </summary>
+        public EventStoreBuilder()
+            : this(new EventStoreConfiguration())
+        {
+            DockerResourceConfiguration = Init().DockerResourceConfiguration;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventStoreBuilder" /> class.
+        /// </summary>
+        /// <param name="dockerResourceConfiguration">The Docker resource configuration.</param>
+        private EventStoreBuilder(EventStoreConfiguration dockerResourceConfiguration)
+            : base(dockerResourceConfiguration)
+        {
+            DockerResourceConfiguration = dockerResourceConfiguration;
+        }
+
+        /// <inheritdoc />
+        protected override EventStoreConfiguration DockerResourceConfiguration { get; }
+
+
+        /// <inheritdoc />
+        public override EventStoreContainer Build()
+        {
+            return new EventStoreContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+        }
+
+        /// <inheritdoc />
+        protected override EventStoreBuilder Init()
+        {
+            return base.Init()
+                .WithImage(EventStoreImage)
+                .WithPortBinding(EventStorePort, true)
+                .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
+                .WithEnvironment("EVENTSTORE_CLUSTER_SIZE", "1")
+                .WithEnvironment("EVENTSTORE_RUN_PROJECTIONS", "All")
+                .WithEnvironment("EVENTSTORE_START_STANDARD_PROJECTIONS", "true")
+                .WithEnvironment("EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP", "true")
```

Comment:
In the docs they mention that `EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP` is [deprecated](https://developers.eventstore.com/server/v20.10/#http). Do we need it?

Reply:
Yes, We can remove it, doc says it's disabled by default.